features/networking/ovn.feature

```diff
@@ -0,0 +1,41 @@
+Feature: OVN related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-29954
+  @admin
+  @destructive
+  Scenario: Creating a resource in Kube API should be synced to OVN NB db correctly even post NB db crash too
+    Given the env is using "OVNKubernetes" networkType
+    Given I have a project
+    And I obtain test data file "networking/list_for_pods.json"
+    When I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(0).name` is stored in the :pod1_name clipboard
+    And evaluation of `pod(1).name` is stored in the :pod2_name clipboard
+    #Checking whether Kube API data is synced on OVN NB db which in this case are couple of pods
+    Given I store the ovnkube-master "north" leader pod in the clipboard
+    And evaluation of `pod.ip_url` is stored in the :OVN_NB_leader_IP clipboard
```

Comment:
@anuragthehatter variable and symbol guide line for ruby https://github.com/rubocop-hq/ruby-style-guide#snake-case-for-symbols-methods-and-variables states that we should use snake_case...probably need to de-uppercase for all of these variables.

Reply:
oops, thats right. Apparently I put some emphasis on networkType (OVN) and its database(NB)